### DISCUSSION TITLE
EntityGenerator does not support MongoDB

### DISCUSSION THREAD

**Discussion by ShenTiger** [mikro-orm/mikro-orm#5051] [General]
I want to generate entities from the existing database schema, but I found that EntityGenerator does not support MongoDB.

`D:\01-Project\simview\apps\server\node_modules\@mikro-orm\core\platforms\Platform.js:279
        throw new Error(`${this.constructor.name} does not support EntityGenerator`);
              ^
Error: MongoPlatform does not support EntityGenerator
    at MongoPlatform.getEntityGenerator (D:\01-Project\simview\apps\server\node_modules\@mikro-orm\core\platforms\Platform.js:279:15)
    at MikroORM.getEntityGenerator (D:\01-Project\simview\apps\server\node_modules\@mikro-orm\core\MikroORM.js:159:42)
    at D:\01-Project\simview\apps\server\generate-entities.ts:22:25
    at processTicksAndRejections (node:internal/process/task_queues:95:5)`

 So I want to find a way to solve this problem. Do you have any good suggestions? 
Please advise, thank you very much!

**Comment by B4nan** [comment]
There is no schema in mongo, we could only do things based on the first/last item in the database, and it would be still extremely limited and error-prone.

So my suggestion is to create entities by hand, its not that hard and you do it only once anyway.
